app/ts/McdaApp/Workspace/CurrentTab/Preferences/EquivalentChange/EquivalentChangeUtil.test.ts

```diff
@@ -134,4 +138,29 @@ describe('equivalentChangeUtil', () => {
       expect(result).toEqual(expectedResult);
     });
   });
+
+  describe('getEquivalentChangeByThreshold', () => {
+    it('should return equivalent change initialized using threshold values', () => {
+      const thresholdValuesByCriterion: Record<string, number> = {
+        refId: 1,
+        notRefId: 15
+      };
+      const prefs: TPreferences = [
+        {criteria: ['refId', 'notRefId']} as IExactSwingRatio
+      ];
+      const state = {
+        thresholdValuesByCriterion,
+        prefs
+      } as IScenarioState;
+      const scenario = {state} as IMcdaScenario;
+      const bounds: [number, number] = [5, 15];
```

Comment:
minor niggle: I got slightly confused by re-use of the specific number 15 here, looking for a connection to the ID

Reply:
nvm, I'm misreading